hack/install-etcd.sh

```diff
@@ -21,7 +21,40 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+LOG_DIR=${LOG_DIR:-"/tmp"}
+
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::etcd::install
+if [ $# -lt 1 ] ; then
```

Comment:
Can you add manual about how to use the script?

Reply:
Added in the comments

Reply:
Added